plugins/foreign-db-extractor/rollup.config.mjs

```diff
@@ -0,0 +1,84 @@
+import commonjs from '@rollup/plugin-commonjs'
+import json from '@rollup/plugin-json'
+import resolve from '@rollup/plugin-node-resolve'
+import terser from '@rollup/plugin-terser'
+import typescript from '@rollup/plugin-typescript'
+import { dts } from 'rollup-plugin-dts'
+import peerDepsExternal from 'rollup-plugin-peer-deps-external'
+
+import dotenv from 'dotenv'
+dotenv.config()
+
+const PROD = process.env.NODE_ENV === 'production'
+if (!PROD) {
+  console.log('Not in production mode - skipping minification')
+}
+
+const external = ['@flatfile/api', '@flatfile/listener', 'mssql']
+
+function commonPlugins(browser, umd = false) {
+  return [
+    !umd
+      ? peerDepsExternal({
+          includeDependencies: true,
```

Comment:
Should we use the same rollup config from the `recordHook` + `autocast` plugins here?